components/automate-ui/src/app/pages/project/list/project-list.component.ts

```diff
@@ -140,7 +142,12 @@ export class ProjectListComponent implements OnInit, OnDestroy {
     this.creatingProject = true;
     const project = {
```

Comment:
Would really urge having this strongly typed, perhaps:
```suggestion
    const project: ProjectPayload = {
```
That same interface definition (`ProjectPayload`) should then also be used in the signature of the `CreateProject` action used on L152. Right now, its constructor uses on-the-fly typing:
```
constructor(public payload: {id: string, name: string, skip: boolean}) { }
```
which would be better served as:
```
constructor(public payload: ProjectPayload) { }
```